cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -419,5 +419,53 @@ CUDA_HOST_DEVICE_CALLABLE constexpr decltype(auto) type_dispatcher(cudf::data_ty
   }
 }
 
+namespace {
```

Comment:
Don't use an anonymous namespace in a header.

http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-unnamed